lib/mixlib/install/options.rb

```diff
@@ -28,11 +28,20 @@ class InvalidOptions < ArgumentError; end
       ARTIFACTORY_CHANNELS = [:unstable]
       ALL_SUPPORTED_CHANNELS = OMNITRUCK_CHANNELS + ARTIFACTORY_CHANNELS
       SUPPORTED_PRODUCT_NAMES = %w[chef chefdk]
-      SUPPORTED_OPTIONS = [:channel, :product_name, :product_version,
-                           :platform, :platform_version, :architecture]
+      SUPPORTED_SHELL_TYPES = [:ps1, :sh]
+      SUPPORTED_OPTIONS = [
+        :architecture,
+        :channel,
+        :platform,
+        :platform_version,
+        :product_name,
+        :product_version,
+        :shell_type
```

Comment:
Let's add a basic spec for `shell_type` to `options_spec` to catch possible regressions in the future.